### DISCUSSION TITLE
Recilcaje de IN-X-RANGE para IN-CIRCLE.

### DISCUSSION THREAD

**Discussion by Cristian1754** [IIC2133-PUC/2024-1#92] [Tarea 1] [answered]
Una pregunta sobre la complejidad de la ultima parte, si es que logro que IN-X-RANGE tenga complejidad de busqueda O(log(n)), en teoria podria aplicar el mismo arbol (con las hojas ordenadas de manera distinta) para poder hacer la busqueda de IN-CIRCLE (solo en el eje x) pero por cada elemento encontrado comprobar si es que el atrubuto "y" esta dentro del circulo, lo cual deberia seguir siendo O(log(n)), ya que hago el mismo procesamiento que IN-X-RANGE. Sin embargo esta idea no sale en la capsula y me entro la duda de si se puede hacer.

Saludos!

**Comment by benjavicente** [comment, marked as answer]
Si se puede 👀
